### PR TITLE
Logging out each webpack asset

### PR DESCRIPTION
**Pull request by thomasdigby**
#### What have you done
A minor update to the console output from the client webpack build

#### Why have you done it
The current output hid the vendor bundle size, it now logs all compiled assets also covering chunks from code-splitting

#### Testing carried out to prevent breaking changes
Tested locally, tests all passing

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -36,12 +36,30 @@ export default class Client {
...
     // run callback
     if (typeof this.opts.onComplete === 'function')
       this.opts.onComplete()
   }
+
+  logAssets (stats) {
+    // log out name/size with a 4 space indent
+    const printAsset = asset =>
+      asset.chunkNames.length > 0 && `
+    ${asset.name} ${bytes(asset.size)}`
```
Comment (mathewhawley) [inline src/client/index.js]: Is there a cleaner way to write this statement? Was a little thrown by the line-breaks in this template string...maybe the string sits on the same line as the bool check?

File: src/client/index.js [hunk 2]
```diff
@@ -36,12 +36,30 @@ export default class Client {
...
+    const assets = stats.assets
+    // if multiple assets
+    if (assets.length > 1) {
+      // break onto new lines and loop through each
+      return assets
+        .map(printAsset)
```
Comment (mathewhawley) [inline src/client/index.js]: Nice use of `.map` ;)

File: src/client/index.js (hunk 1 above)
Comment (thomasdigby) [inline src/client/index.js]: The line-break/spacing is necessary for the output, maybe I'll `/n/t` instead

File: src/client/index.js (hunk 2 above)
Comment (mathewhawley) [inline src/client/index.js]: Just thinking `printAsset` is a little misleading of a name? From looking at the function it's just building a string of sorts – maybe `buildStat` or something? The parent `logAssets` function kinda does the 'printing'? Subjective I know, fine to leave if you're happy with it

File: src/client/index.js (hunk 1 above)
Comment (mathewhawley) [inline src/client/index.js]: Ah ok – yea whatever works – if it's needed then all good

**Pull request merged**